PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-guidance/src/main/resources/flow-schema/tasks.json

```diff
@@ -260,6 +260,29 @@
           }
         }
       }
+    },
+    {
+      "additionalProperties": false,
```

Comment:
What's the usage of this parameter? Do we need to udpate the config/mapping model?

Reply:
> What's the usage of this parameter? 
https://json-schema.org/understanding-json-schema/reference/object.html#additionalproperties
> Do we need to udpate the config/mapping model?
no